### ISSUE TITLE
As a researcher I want to access my user storage across my Windows and Linux virtual machines without additional configuration.

### ISSUE THREAD

**Issue by daltskin** [microsoft/AzureTRE#1566]

**Comment by martinpeck** [comment]
@marrobi is this an accurate reading of the requirements...?

- Researchers, within a workspace, should be given a folder that is clearly "theirs". e.g. the folder name is based upon their username, or the folder is their home folder, or something similar. In my case, that folder might be named `martinpeck`
- This folder should be present when they first attach to a VM within a workspace
- This folder is shared across all VMs within that workspace
- This folder is shared across Windows and Linux VMs within that workspace
- Other users within the workspace have their own folders
- Other users within the workspace might be able to traverse the filesystem and see the contents of another user's folder. Once in another user's folder, that user can manipulate (create/read/update/delete) files.
- These user folders are not shared across workspaces

**Comment by martinpeck** [comment]
sorry...should have numbered that list to make it easier to discuss (done)

**Comment by marrobi** [comment]
Correct, add to no 3 - "shared across all **their** VMs"

**Comment by marrobi** [comment]
On the file system previously used a users object id as the azure file share name.

**Comment by ross-p-smith** [comment]
I'm assuming that a user does not know their objectid though? Do you mean their AAD objectid

**Comment by tamirkamara** [comment]
If we mount the same shared storage to each VM, can't users create their own folder structure there?

**Comment by martinpeck** [comment]
I agree @tamirkamara. If they have shared storage, the folder structure they create (whether it be based upon usernames or otherwise) is up to them.

@marrobi other than the convenience of knowing "when I join a workspace I can assume there's a folder with my name on it"  is there some other reason we need to do this?

**Comment by marrobi** [comment]
Two scenarios that need covering. Although wouldn't priotise this, has been asked for, but not a blocker.

1. I do some work on a VM. I store that work in my home folder. I resize that VM. I lose my work. Hence some shared storage should be mounted to the common profile directories - home and desktop spring to mind.

2. Although data does not need to be secured privately, I want to have my own space where it is not easy for other users to move/delete my files.

First one is more pressing as think it will be an issue before long.
